controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -203,8 +209,7 @@ module.exports = async (req, res) => {
                             a.only_received_dm_from_user_following,
                             a.is_backdoor_user,
                             a.followers_count,
-                            a.karma_score,
-                            b.topic_id
+                            a.karma_score
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `b.topic_id` field has been removed from the SQL query. If this was intentional and the field is not needed, then it's fine. However, if it was removed by mistake, you should add it back.
```diff
+                            b.topic_id,
                             a.karma_score
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->